src/easylink/cli.py

```diff
@@ -58,7 +58,7 @@ def easylink():
 
 
 @easylink.command()
-@pass_shared_options
+@_pass_shared_options
```

Comment:
made these private b/c I didn't want it showing up in the api docs. Happy to discuss